css/index.css

```diff
@@ -71,6 +71,45 @@ p {
     margin: 0 auto;
```

Comment:
Great job creating seperate CSS files for the about/index.html page, it keeps your code more organized and readable.